### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by katcipis**
Initial issue templates for features/bugs

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+A clear and concise description of what you expected to happen.
+
+**Log Output**
+Add logs from Terramate to help debug your problem.
+
+**Environment (please complete the following information):**
+ - OS: [e.g. Linux]
+ - OS Version [e.g. Ubuntu 20.04]
+ - Terramate Version [e.g. 0.1.0]
+
```
Comment (i4ki) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: The git version is also important.

Comment (katcipis) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: well remembered my dude :100:

**Review by i4ki — APPROVED** [review]
lgtm

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**